content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Nested Queries
+further_reading:
+- link: "/dashboards/querying/"
+  tag: "Documentation"
+  text: "Dashboard Querying"
+---
+
+## Overview
+
+{{< callout url="#" btn_hidden="true" header="Join the Preview!">}}
+  This feature is in Preview. Please reach out to Customer Success to get this feature enabled. 
+{{< /callout >}}  
+
+Every metric query in Datadog consists of two layers of aggregation by default (quick refresher [here][5]).
+Nested queries allows you to reuse the results of a previous query in a subsequent one.
```

Comment:
```suggestion
By default, every metric query in Datadog consists of two layers of aggregation. Nested queries allows you to reuse the results of a previous query in a subsequent one.
```

Reply:
I removed the `(quick refresher [here][5])` since it's repeated in the next section.